src/main/scala/com/danielasfregola/twitter4s/entities/User.scala

```diff
@@ -36,6 +36,7 @@ final case class User(blocked_by: Boolean = false,
                 profile_image_url: ProfileImage,
                 profile_image_url_https: ProfileImage,
                 profile_link_color: String,
+                profile_location: Option[String],
```

Comment:
Couldn't find any non-null representation of this either....so I guess string will do for now!